src/btree/pagefile.ts

```diff
@@ -0,0 +1,25 @@
+import { RangeResolver } from "../resolver";
+
+const MAX_FREE_PAGE_INDICIES = 512;
+const PAGE_SIZE_BYTES = 4096;
+
+export class PageFile {
```

Comment:
I don't think we need this class anymore, since we have `getPage()`

But I may be incorrect